lecture_08_object_model/hw/hw1.py

```diff
@@ -0,0 +1,24 @@
+class KeyValueStorage(dict):
+    """Key Value Storage class which acts like a dict"""
+
+    def __init__(self, path: str):
+        super().__init__()
+        with open(path) as f:
+            for line in f:
+                key, value = line.strip().split('=')
+
+                if key in KeyValueStorage.__dict__ or key in dict.__dict__:
+                    continue
```

Comment:
Плохая идея - думаю хранить данные как реальные атрибуты класса не лучшая идея.
Если я захочу в файле создать например такие значения:
```
keys=54
items=MY_ITEMS
```

Reply:
Так у нас по заданию: «In case of attribute clash existing built-in attributes take precedence.»
То есть приоритет должен отдаваться встроенным значениям.
Значит storage['keys'] должно вернуть `<built-in method keys of KeyValueStorage object at...`

Я, правда, в случае коллизии  лучше бы поднимал ошибку ValueError при попытке инициализации экземпляра класса